tests/framework/config/cluster.go

```diff
@@ -14,15 +14,23 @@
 
 package config
 
-import "time"
+import (
+	"time"
+)
 
 type TLSConfig string
+type ClusterVersion string
 
 const (
 	NoTLS     TLSConfig = ""
 	AutoTLS   TLSConfig = "auto-tls"
 	ManualTLS TLSConfig = "manual-tls"
 
+	CurrentVersion      ClusterVersion = ""
+	MinorityLastVersion ClusterVersion = "minority-last-version"
+	QuorumLastVersion   ClusterVersion = "quorum-last-version"
+	LastVersion         ClusterVersion = "last-version"
```

Comment:
We shouldn't have this option `LastVersion`, because it doesn't make sense to test previous version in `main` branch CI.

Reply:
Upgrade tests start from LastVersion.